drivers/cc110x/cc110x.c

```diff
@@ -175,3 +175,34 @@ int cc110x_set_channel(cc110x_t *dev, uint8_t channel)
     dev->netdev.event_callback(&dev->netdev, NETDEV_EVENT_FHSS_CHANGE_CHANNEL);
     return 0;
 }
+
+int cc110x_wakeup(cc110x_t *dev)
```

Comment:
I'd rather have this called `cc110x_power_on()` to match the name of `cc110x_power_off()`. Maybe the existing `cc110x_power_on()` function could be renamed to `cc110x_low_level_power_on()`, or `cc110x_init_chip()`, or ...?